ntbs-service/DataMigration/MigrationRepository.cs

```diff
@@ -77,9 +79,10 @@ public async Task MarkNotificiationsAsImportedAsync(IEnumerable<Notification> no
 
                 var importedAt = DateTime.Now.ToString("s");
 
+                var query = string.Format(InsertImportedNotificationsQuery, _notificationHelper.GetImportedNotificationsTableName());

```

Comment:
I'm tempted to suggest that we should ask the notification helper for the fully ready query, rather than calling `stringFormat` here.
```suggestion
                var query = _importHelper.InsertImportedNotificationsQuery;
```
That way both queries that have the table name varied live in the same place and we're containing this dodgy style to one class only